pages.de/osx/afinfo.md

```diff
@@ -0,0 +1,29 @@
+# afinfo
+
+> Audiodatei-Metadaten-Parser für OS X.
+> Eingebauter Befehl von OS X.
+> Weitere Informationen: <https://keith.github.io/xcode-man-pages/afinfo.1.html>.
+
+- Anzeige von Informationen zu einer bestimmten Audiodatei:
+
+`afinfo {{pfad/zur/datei}}`
```

Comment:
```suggestion
`afinfo {{pfad/zu/datei}}`
```

Reply:
Which one is the preferred one? Both exists in other German translations.

Reply:
Our [Common Arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md) only mention `pfad/zu/` and from what I can remember _zum_/_zur_/etc is informally discouraged (hence much less common), but I don't think we have a rule regarding this either.

Reply:
Thanks for the clarification. I might go through the other files and fix them accordingly.